benchmarks/benchmark_serving.py

```diff
@@ -599,12 +599,14 @@ def main(args: argparse.Namespace):
         max_output_length=args.max_output_length,
     )
 
-  if args.warmup_first:
-    print("Warm up start:")
-    if args.full_warmup:
-      warmup_requests = input_requests
-    else:
-      warmup_requests = list(sample_warmup_requests(input_requests)) * 2
+  warmup_requests = None
+  if args.warmup_mode == "full":
```

Comment:
Should we make it to `num_decode_slots * (len([16, 32, 64, 128, 256, 512, 1024]) = batch_size * num_chips * 7` to ensure the server is completely warmup?

Reply:
Let's follow up with diff PR for correct/fixing different warmup mode